src/api/operator/numpy/np_matrix_op.cc

```diff
@@ -470,4 +470,66 @@ MXNET_REGISTER_API("_npi.diag_indices_from")
   *ret = ndoutputs[0];
 });
 
+MXNET_REGISTER_API("_npi.repeat")
+.set_body([](runtime::MXNetArgs args, runtime::MXNetRetValue* ret) {
+  using namespace runtime;
+  const nnvm::Op* op = Op::Get("_npi_repeat");
+  nnvm::NodeAttrs attrs;
+  op::RepeatParam param;
+  param.repeats = args[1].operator int();
```

Comment:
So `_npi_repeat` itself does not support `repeats` when it is array of ints, right?